### DISCUSSION TITLE
Mutation Reports Name

### DISCUSSION THREAD

**Discussion by MaximeDan** [stryker-mutator/stryker-net#2242] [Ideas]
Hello,

Currently I'm using Stryker on a big Solution with many projects. I set up Stryker in my Azure pipeline and make it run on each project. My issue is that I have different reports, but with this generic name "mutation-report-1" "mutation-report-2".
I'm thinking about changing the default value ine "ReportFileNameInput.cs" but i wonder if this change could be interesting for everyone ?

**Comment by rouke-broersma** [comment]
@MaximeDan you can set the report file name through configuration: https://stryker-mutator.io/docs/stryker-net/configuration/#report-file-name-string. This will however not change the 'mutation-report-1' etc in the azure devops extension.

The extension needs to be changed, there is already an issue for it with a suggestion for how it could be fixed: https://github.com/stryker-mutator/azure-devops-mutationreport-publisher/issues/730

**Reply by MaximeDan** [reply]
Great thank you !
